system_test/aest_sync_SUITE.erl

```diff
@@ -384,8 +390,43 @@ net_split_recovery(Cfg) ->
 
     ok.
 
+sync_speed(Cfg) ->
+    InitialNodes = [node_name(I) || I <- lists:seq(1, 4)],
+    NextNodes = [node_name(I) || I <- lists:seq(5, 6)],
+    Height = 1000,
+
+    Spec = #{
+        backend => aest_docker,
+        source => {pull, "aeternity/epoch:v0.11.1"},
+        mine_rate => 100
+    },
+
+    InitialNodeSpecs = [
+        maps:merge(#{name => N, peers => InitialNodes -- [N]}, Spec)
+        || N <- InitialNodes
+    ],
+    NextNodeSpecs = [
+        maps:merge(#{name => N, peers => InitialNodes}, Spec)
+        || N <- NextNodes
+    ],
+
+    setup_nodes(InitialNodeSpecs ++ NextNodeSpecs, Cfg),
+    [start_node(N, Cfg) || N <- InitialNodes],
+    wait_for_value({height, Height}, InitialNodes, ?MINING_TIMEOUT * Height, Cfg),
+
+    start_node(node_name(5), Cfg),
+    wait_for_value({height, 0}, [node_name(5)], ?STARTUP_TIMEOUT, Cfg),
+    wait_for_value({height, Height}, [node_name(5)], ?SYNC_TIMEOUT * Height, Cfg),
```

Comment:
Putting :local, at this time locally node5 had arrived at height 818

Reply:
Two sets of blocks are a fork?

Reply:
I did not check whether the two sets of blocks were belonging to distinct forks. They could have been well portions of the same fork at different height based only on the logs (for some reason). I did not check the hashes.

Reply:
@lucafavatella I guess this would show if I compare the blocks at height 1000 in the end as well, even if there is no timeout. If what you observed was a fork, what could be the cause? Getting a fork with only 5 nodes doesn't seem so nice...